spec/cicognara/tei_indexer_spec.rb

```diff
@@ -37,6 +37,10 @@
       expect(subject.items.first.n).to eq('1')
     end
 
+    it 'has a title property' do
+      expect(subject.items.first.title).not_to be_nil
+    end
+
```

Comment:
As with an earlier PR that went by, I think you should test that the values are what you expect, e.g.:
```ruby
expected = "Libellus de utilitate et harmonia artium, Antuerpiæ, apud Simonem Cocum"
expect(subject.items.first.title).to eq expected
```